ubr/conf.py

```diff
@@ -38,12 +38,17 @@
 _handler.setLevel(logging.DEBUG)
 _handler.setFormatter(logging.Formatter('%(levelname)s - %(asctime)s - %(message)s'))
 
+_filehandler = logging.FileHandler('ubr.log')
+_filehandler.setFormatter(_formatter)
+_filehandler.setLevel(logging.INFO)
+
 ROOTLOG.addHandler(_handler)
+ROOTLOG.addHandler(_filehandler)
 ROOTLOG.setLevel(logging.DEBUG)
 
 # tell boto to pipe down
-import boto3
-boto3.set_stream_logger('', logging.CRITICAL)
+loggers = ['boto3', 'botocore', 's3transfer']
+[logging.getLogger(nom).setLevel(logging.CRITICAL) for nom in loggers]
```

Comment:
as much as Boto is verbose, `ERROR` should help as a level to track if something goes wrong?